_data/simple-icons.json

```diff
@@ -4901,6 +4901,19 @@
             "hex": "4285F4",
             "source": "https://earth.google.com/web/"
         },
+        {
+            "title": "Google Earth Engine",
+            "hex": "4285F4",
+            "source": "https://earthengine.google.com",
+            "aliases": {
+                "aka": [
+                    "Earth Engine",
+                    "earthengine",
+                    "GEE",
+                    "EE"
+                ]
+            }
```

Comment:
I don't think we need all these aliases. You can probably just get away with the main title :)